typescript/sdk/src/deploy/core/HyperlaneCoreChecker.ts

```diff
@@ -34,10 +24,8 @@ export class HyperlaneCoreChecker<
 
     await this.checkDomainOwnership(chain);
     await this.checkProxiedContracts(chain);
-    await this.checkOutbox(chain);
-    await this.checkInboxes(chain);
-    await this.checkConnectionManager(chain);
-    await this.checkValidatorManagers(chain);
+    await this.checkMailbox(chain);
+    // await this.checkDefaultIsm(chain);
```

Comment:
does this exist yet? should `checkValidatorManager` be changed into this?

Reply:
Yes it should have been, good catch